pandas/core/indexes/base.py

```diff
@@ -1471,6 +1471,41 @@ def is_object(self):
         return is_object_dtype(self.dtype)
 
     def is_categorical(self):
+        """
+        Check if the Index type is categorical.
+
+        The given object must be a pandas DataFrame or pandas Series.
```

Comment:
this only applies to a Series or an Index, IOW it must be 1D

Reply:
Thanks @jreback ! I have changed the exended summary. Is it better now?